scalyr_agent/builtin_monitors/kubernetes_monitor.py

```diff
@@ -3032,7 +3033,7 @@ def __start_docker_logs(self, docker_logs):
         for log in docker_logs:
             if self.__log_watcher:
                 log["log_config"] = self.__log_watcher.add_log_config(
-                    self.__module.module_name, log["log_config"]
+                    self.__module.module_name, log["log_config"], force_add=True
```

Comment:
When I added that change a while back, both @czerwingithub and I agreed it's probably safe to make ``force_add=True`` a default behavior, but to me on the safe side and avoid potential issues, I decided to just set it to True in the Docker monitor where a race / issue was observed.

Having said that, I'm not too surprised that a similar issue exists in the Kubernetes monitor since there is quite a lot of duplicated and similar code in those two monitors.

Reply:
EDIT: Yeah, looks like that the issue was present in the same method (https://github.com/scalyr/scalyr-agent-2/blob/master/scalyr_agent/builtin_monitors/docker_monitor.py#L1259) so very likely same root cause for both issues.